### ISSUE TITLE
Add ability to ignore certain namespaces

### ISSUE THREAD

**Issue by mak-dunkelziffer** [djezzzl/database_consistency#173]
There are cases, where it would be nice to have a more comfortable method of ignoring multiple models or attributes at once in the config.

I have one case where the namespace `DataMigration` contains models that are not used in production and are kept for historic reasons. So I currently have 44 lines of the following in my config:
```yaml
DataMigration::Article
  enabled: false
...
DataMigration::User
  enabled: false
```

I have another case where errors are "inherited" from another gem and I have 13 lines of the following in my config:
```yaml
User:
  thredded_private_posts:
    enabled: false
...
  thredded_post_moderated_records:
    enabled: false
```
Being able to ignore all attributes starting with `thredded_` (per model or even globally) would be nice.

**Comment by djezzzl** [comment]
Hi @mak-dunkelziffer,

Thank you for suggesting this! We can add support for some patterns there. What syntax do you prefer the most?

For example, 'DataMigration::%', `DataMigration::*`, or somehow else.

**Comment by mak-dunkelziffer** [comment]
If it is possible to pass a Ruby regex in the Yaml file, that would probably be the easiest. Then you wouldn't have to implement any custom pattern parsing yourself.

**Comment by djezzzl** [comment]
Sounds fair! I will look into that. 

BTW, would you like to contribute? I would be happy to assist you and release it ASAP.

**Comment by djezzzl** [comment]
Hi @mak-dunkelziffer,

I'm sorry it took me so long to implement this feature. Nevertheless, it's finally done, and if you're still using the gem, you can clean up your `.database_consistency.yml` configuration. 

https://github.com/djezzzl/database_consistency/pull/202 

Released in 1.7.11.

P.S. Fixed `changelog.md` with release commit.

**Comment by djezzzl** [comment]
You can follow this wiki for precise syntax: https://github.com/djezzzl/database_consistency/wiki/regexp-support-in-configuration.
